### ISSUE TITLE
(WCAG 2.1) Lack of Descriptive Legend for Phone Number Input Field Grouping 

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#5526]
OBSERVED RESULTS:

- The phrase "What is your phone number?" is not being used as a legend for the grouping of the "Country" and "Enter Phone number" input fields.
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/8484029d-80fe-4c7e-9f65-adf197d13716)

EXPECTED RESULTS:

- Associate the phrase "What is your phone number?" as a <legend> element to properly label and group the "Country" and "Enter Phone number" input fields.

STEPS TO REPRODUCE:

- Enable screen reader.
- Navigate to [URL](https://demo.civiform.dev/)
- Navigate to and activate the ‘Apply’ for the Comprehensive sample program.
- Navigate to and activate “What is your address?”.
- Navigate to "Country" and "Enter Phone number" input fields keyboard TAB or SHIFT TAB key.
- Observe the on receiving focus screen reader does not announce the group legend.

ENVIRONMENT DETAILS:

- Device(s) (brand, OS version, test key or release key): Windows 10 Enterprise (64-bit),MacOS Ventura Version 13.4
- Screen reader name and version: JAWS 2022.2211.7, NVDA 2022.3, VoiceOver
- Browser name and version: Chrome Version 113.0.5672.93

**Comment by melanie-exygy** [comment]
@dkatzz The screenshot above doesn't seem to match the description of the issue.  Could this be the wrong screenshot?

**Comment by dkatzz** [comment]
> @dkatzz The screenshot above doesn't seem to match the description of the issue. Could this be the wrong screenshot?

You're right - updated to what I think is the correct screenshot.
